starter-repos/app-template/skaffold/render.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Different from the render in ../shared-ci-cd/skaffold/render.yaml,
+# this skaffold-render updates the *DRY* config with the new skaffold image and
+# do not hydrate the resources. The resource hydration is done via `appctl prepare`  
+skaffold-render:
+  stage: render-manifests
+  image: gcr.io/k8s-skaffold/skaffold
```

Comment:
Specify an image tag

Reply:
The previously used skaffold image is actually pretty old. I learnt that the latest skaffold image is more reliable considering that the skaffold is GA and have relatively reliable backward compatibility policy.

Reply:
Adding an image tag makes the pipeline more stable and repeatable. 

This comment is about adding an image tag here. You may also update the skaffold image in shared-cicd to use v1.14 too.